src/dscanner/main.d

```diff
@@ -237,7 +242,21 @@ else
 		if (skipTests)
 			config.enabled2SkipTests;
 		if (report)
-			generateReport(expandArgs(args), config, cache, moduleCache);
+		{
+			switch (reportFormat)
+			{
+				case "":
+				case "dscanner":
+					generateReport(expandArgs(args), config, cache, moduleCache);
+					break;
+				case "sonarQubeGenericIssueData":
+					generateSonarQubeGenericIssueDataReport(expandArgs(args), config, cache, moduleCache);
+					break;
+				default:
+					stderr.writeln("Unknown report format specified, using default format");
```

Comment:
maybe to avoid code duplication you could write
```d
default:
	stderr.writeln("Unknown report format specified, using dscanner format");
	goto case;
case "":
case "dscanner":
	...
```

Also it should log "using dscanner format"

Reply:
fixed